app/workers/comments/calculate_score_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Comments
+  class CalculateScoreWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority
```

Comment:
@mstruve do jobs have a default retry set?

Reply:
By default it will retry 25 times over 21 days - https://github.com/mperham/sidekiq/wiki/Error-Handling, which seems like it should be more than enough for this worker?

Reply:
What @VegaFromLyra said 😃

The reason we set the retries to 10 for a lot of jobs is that we don't necessarily want them to spend a month retrying. I think 4 hours is enough bc if an incident happens we likely can resolve it in under 4 hours and not have the jobs fail. But this is totally fine. I might update our default at some point.